src/Pact/Analyze/Types/Eval.hs

```diff
@@ -528,6 +523,12 @@ purelyReachable = latticeState.lasPurelyReachable.sbv2S
 maintainsInvariants :: Lens' (AnalyzeState a) (TableMap (ZipList (Located SBool)))
 maintainsInvariants = latticeState.lasMaintainsInvariants
 
+guardPasses :: Lens' (AnalyzeState a) (SFunArray Guard Bool)
+guardPasses = latticeState.lasGuardPasses
+
+guardPass :: S Guard -> Lens' (AnalyzeState a) (S Bool)
```

Comment:
Not in love with the `guardPass` / `guardPasses` naming.

Reply:
Same. Will think on this